integrations/sklearn/MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include versioneer.py
+include neptune/alpha/integrations/sklearn/impl/_version.py
```

Comment:
Shouldn't it be new instead of alpha already?

Reply:
I followed template provided by @HubertJaworski. Should I correct it @HubertJaworski ?

Reply:
fixed